sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md

```diff
@@ -88,6 +88,15 @@ client = FooServiceClient(
 response = client.get_foo_properties()
 ```
 
+### Pipeline client configurations
+
+| Parameters | Description |
+| --- | --- |
+| `pipeline` | While `PipelineClient` will create a default pipeline, users can opt to use their own pipeline by passing in a `Pipeline` object. If passed in, the other configurations will be ignored.  |
+| `config` | While `PipelineClient` will create a default `Configuration`, users can opt to use their own configuration by passing in a `Configuration` object. If passed in, it will be used to create `Pipeline` object. |
+| `transport` | While `PipelineClient` will create a default `RequestsTransport`, users can opt to use their own transport by passing in a `RequestsTransport` object. If it is omitted, `PipelineClient` will honor other transport customization described in [Transport](#transport). |
```

Comment:
tiny nit:  "If it is omitted, `PipelineClient` will honor other the described transport customizations"